### PR TITLE
Add test to check numa of created namespace

### PR DESCRIPTION
**Pull request by harish-24**
Patch adds test to validate numa of created namespace through ndctl and its sys interface

Signed-off-by: Harish <harish@linux.ibm.com>

### CODE REVIEW

File: memory/ndctl.py [hunk 1]
```diff
@@ -432,6 +432,27 @@ def test_check_numa(self):
...
+        self.plib.disable_namespace()
+        self.plib.destroy_namespace()
+        if (not self.plib.is_region_legacy(region)):
+            for _ in range(3):
+                self.plib.create_namespace(
+                    region=region, mode='fsdax', size='128M')
+
+        namespaces = self.plib.run_ndctl_list('-N -r %s' % region)
+        if not os.path.exists('/sys/bus/nd/devices/namespace0.0/numa_node'):
+            self.fail("Numa node entries not found!")
```
Comment (PraveenPenguin) [inline memory/ndctl.py]: is this should cancel or fail as if basic requirement not there test should cancel that's what I feel

Comment (harish-24) [reply memory/ndctl.py]: It is a bug if we do not find the entry after enabling namespace.

**Comment by harish-24** [comment]
```
JOB LOG    : /home/harish/avocado-fvt-wrapper/results/job-2020-03-11T11.34-ff19e2d/job.log
 (1/1) memory/ndctl.py:NdctlTest.test_check_ns_numa: PASS (2.55 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 2.65 s
```
Legacy h/w:
```
 (1/1) memory/ndctl.py:NdctlTest.test_check_ns_numa: FAIL: Numa mismatch between ndctl and sys interface (10.41 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 10.70 s
```

File: memory/ndctl.py (hunk 1 above)
Comment (PraveenPenguin) [reply memory/ndctl.py]: ah sure

**Pull request merged**